### DISCUSSION TITLE
[A-L] (12 Giugno 2023) Esercizio 5

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#390] [Esami A-L]
Sia $\Large A = \\{1,\ 2,\ 3,\ 4,\ 5,\ 6\\}$ e $\Large R$ una relazione binaria su $\Large A$.

Consideriamo il linguaggio proposizionale $\Large \mathcal{L}$ contenente una variabile $\Large r_{(a,b)}$ per ogni scelta di $\Large a, b \in A$ (non necessariamente distinti).

Il significato intuitivo della variabile $\Large r_{(a,b)}$ è $\Large (a, b) \in R$ (che leggiamo: $\Large a$ è in relazione con $\Large b$).

Formalizzare le proprietà seguenti nel linguaggio $\Large \mathcal{L}$:

1. 3 è in relazione con 5 e con 6. [1 punto]

2. 4 è in relazione solo con 2 e con 6. [1 punto]

3. La relazione $\Large R$ è simmetrica. [1 punto]

4. La relazione $\Large R$ non è riflessiva. [1 punto]

5. Esistono tre elementi di $\Large A$, siano $\Large a,\ b,\ c$ tali che $\Large (a, b) \in R,\ (b, c) \in R$ e $\Large (a, c) \notin R$. [1 punto]

(Suggerimento: una quantificazione universale (per ogni) su un insieme finito si esprime con una congiunzione, una quantificazione esistenziale (esiste) su un insieme finito si esprime con una disgiunzione.)

**Comment by cek753** [comment]
1. $r(3,5)\land r(3,6)$
2. $r(4,2) \land r(4,6) \land \bigwedge_{x \in \{1, 3, 4, 5\}} \lnot r(4,x)$
3. $\bigwedge_{a,b \in A}  r(a,b) \rightarrow r(b,a)$
4. $\bigvee_{a \in A} \lnot r(a,a)\$
5. $\bigvee_{a,b,c \in A}r(a,b)\land r(b,c) \land \lnot r(a,c)$
